File: 1 - Fundamentos/bloco_4/dia_4/part_2/exercicio_4.js
function maiorPalavra(arr) {
  let word = arr[0];

  for (let index in arr) {
    if (word.length < arr[index].length) {
      word = arr[index];
    }
  }
  return word;
}

console.log(
  maiorPalavra(["José", "Lucas", "Nádia", "Fernanda", "Cairo", "Joana"])
);

File: 4 - CS/Bloco_32/exercicios/practice/exercicio_4.py
def find_higher_name(names):
    higher_name = names[0]
    for name in names:
        if len(name) > len(higher_name):
            higher_name = name
    return higher_name


array_names = ["José", "Lucas", "Nádia", "Fernanda", "Cairo", "Joana"]

print(find_higher_name(array_names))
